doc/adding_to_an_adapter.md

```diff
@@ -1,120 +1,237 @@
+use of multiple credential.
+
+> ### VMware Aria Operations Integration SDK library
+> ```python
+> def get_adapter_definition() -> AdapterDefinition:
+>     definition = AdapterDefinition("MyAdapter", "My Adapter")
+>
+>     credential = definition.define_credential_type("my_credential_type", "Credential")
+>     credential.define_string_parameter("user_name", "User Name")
+>     credential.define_password_parameter("user_password", "Password", required=False)
+>     # Second set of credentials
+>     second_credential = definition.define_credential_type("my_second_credential_type", "Second Credential")
+>     second_credential.define_string_parameter("second_user_name", "Second User Name")
+>     second_credential.define_password_parameter("second_user_password", "Second Password", required=False)
+> ```
+>> Note: The `AdapterDefinition` will generate a describe.xml equivalent to the one below
+> ### describe.xml
+> ```xml
+> <AdapterKind xmlns="http://schemas.vmware.com/vcops/schema" key="my_adapter" nameKey="1" version="1">
+>   <CredentialKinds>
+>       <CredentialKind key="my_credential_type" nameKey="2" >
+>         <CredentialField required="true" dispOrder="0" enum="false" key="user_name" nameKey="3" password="false" type="string"/>
+>         <CredentialField required="false" dispOrder="1" enum="false" key="user_password" nameKey="4" password="true" type="string"/>
+>       </CredentialKind>
+>       <CredentialKind key="my_second_credential_type" nameKey="2" >
+>           <CredentialField required="true" dispOrder="0" enum="false" key="second_user_name" nameKey="3" password="false" type="string"/>
+>           <CredentialField required="false" dispOrder="1" enum="false" key="second_user_password" nameKey="4" password="true" type="string"/>
+>       </CredentialKind>
+>   </CredentialKinds>
+>   <ResourceKinds>
+>       <ResourceKind key="MyAdapter_adapter_instance" nameKey="2" type="7" credentialKind="my_credential_type,my_second_credential_type"/>
+>   </ResourceKinds>
+> </AdapterKind>
+> ```
+>> In the describe.xml credential kind keys are separated by a comma.
+
+Once the credential is defined in the [object model](#defining-an-adapter-and-adapter-instance-in-the-object-model), it can be used in the adapter code.
+> Note: If there are any existing connections used by the [`mp-test`](mp-test.md) tool before the credential was created
+> or updated, these will need to be deleted or updated.
+
+Once an adapter instance is defined, any credential defined in the [object model](#defining-an-adapter-and-adapter-instance-in-the-object-model)
+will be prompted to the user when creating an account in VMware Aria Operations on the `Data Sources` &rarr; `Integrations`
+page. After the account has been created, credential fields will be available `AdapterInstance` object passed to the
+`collect`, `test`, and `get_endpoints` methods.(See [Creating an Adapter Instance](#creating-an-adapter-instance).)
```

Comment:
May want to mention `get_credential_type` for determining which type of credential the user has supplied, if there are more than one